dictionary_client/commands.py

```diff
@@ -32,11 +34,13 @@ def disconnect_command():
 
 @dictionary_server_command
 def define_word_command(word, db="*"):
-    return f"DEFINE {db} {word}"
+    word = format_word(word)
+    return f"Define {db} {word}"
 
 
 @dictionary_server_command
 def match_command(word, db="*", strategy="exact"):
+    word = format_word(word)
     return f"MATCH {db} {strategy} {word}"
```

Comment:
A thought - maybe we could add a `Word` class, move the `format_word` helper onto a method there, and call it in `Word.__str__`. I think the way you've implemented this is great though, so I'll leave that to your taste.

Reply:
I like this more